lib/runtime/console.js

```diff
@@ -17,8 +17,8 @@ const { changeprompt, changemodule, fullpath } =
 
 const isWindows = process.platform === 'win32'
 const uriRegex = isWindows ?
-  /(@ ([^\s]+)\s(.*?)\:(\d+)|((([a-zA-Z]:|\.\.?|\~)|([^\0<>\?\|\/\s!$`&*()\[\]+'":;])+)?((\\|\/)([^\0<>\?\|\/!$`&*()\[\]+'":;])+)+\.[^\0<>\?\|\/\s!$`&*()\[\]+'":;]+)(\:\d+)?)/ :
-  /(@ ([^\s]+)\s(.*?)\:(\d+)|(((\.\.?|\~)|([^\0\s!$`&*()\[\]+'":;\\])+)?(\/([^\0\s!$`&*()\[\]+'":;\\])+)+)(\:\d+)?)/
+  /(@ ([^\s]+)\s(.*?)\:(\d+)|((([a-zA-Z]:|\.\.?|\~)|([^\0<>\?\|\/\s!$`&*()\[\]+'\":;])+)?((\\|\/)([^\0<>\?\|\/\n\r!$`&*()\[\]+'\":;])+)+(\.|\\|\/)[^\0<>\?\|\/\s!$`&*()\[\]+'\":;]+)(\:(\d+))?)/ :
```

Comment:
why do we need `\` before `"` only on Windows ?

Reply:
Your're right these are not necessary here as they are artifact from me testing Julia first. It works either way, but I'll remove them for consistency.

Thanks.